### DISCUSSION TITLE
How to publish on `JSR` and `NPM`

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [jsr-io/jsr#397] [Q&A]
My question is how to publish a ts transpired code to basic ESM to JSR and npm ?

**Comment by tflanagan** [comment]
Interested in using JSR, but not really looking to have additional DX overhead when one of the major appeals to JSR is reduced DX overhead.

An integration with NPM publish would be the tipping point in using JSR over NPM for me and I imagine many other existing NPM package maintainers.

I don't quite see any docs on this, and the demo video from yesterday didn't touch on this.

**Reply by AugustinMauroy** [reply]
https://github.com/AugustinMauroy/typescript-package-template
